src/handlers/misc/Roles.ts

```diff
@@ -0,0 +1,31 @@
+import { Message, GuildMemberRoleManager, MessageEmbed } from 'discord.js'
+import { User } from '../../entity/User'
+import AbstractHandler from '../abstract/AbstractHandler'
+import { Command } from '../../entity/Command'
+import { buildEmbed } from "../../util/embeds"
+import config from '../../config/config'
+
+export default class RoleHandler extends AbstractHandler {
+    public constructor() {
+        super(false, false, false, false)
+    }
+
+    protected async handler(user: User, cmd: Command, msg: Message): Promise<any> {
+        const userRoles: GuildMemberRoleManager = await msg.member.roles
+        const raconteur = config.roles.raconteur
+        let roleAssigned = false
+
+        if (await userRoles.cache.has(raconteur)) {
+            await userRoles.remove(raconteur)
+        } else {
+            await userRoles.add(raconteur)
```

Comment:
Don't we want to guard this to see if they are Janissary or above?

Reply:
Jakk told me that he will be limiting it with channel permissions. So no, that won't needed.